documentation/smg/10_build_release_process.md

```diff
@@ -46,21 +46,24 @@ platforms, and the code has been reviewed, the PR can be merged.
 The Jenkins build artifacts are are not accessible outside of STFC.
 End users will be directed to GitHub to access packaged releases.
 
-A 'deploy' pipeline will be created
-([Horace #73](https://github.com/pace-neutrons/Horace/issues/73)).
+Release pipelines have been created on ANVIL,
+these are to be used to create release candidates.
+
+Deploy pipelines have also been created to push release candidates to GitHub.
 
 This will:
 
-- tag the `master` branch with the release number (`Rm.n.o`)
-- copy the built artifact to GitHub
+- tag the Git revision with the release number (`vm.n.o`)
```

Comment:
What is the reasoning behind this change?

Reply:
GitHub suggests using, for example, `v3.4.1`, when creating releases. I tend to agree, `v` for version is more standard than `R` for release - at least in my experience.